src/MarDyn.cpp

```diff
@@ -160,8 +161,8 @@ int main(int argc, char** argv) {
 	if( options.is_set_by_user("logfile") ) {
 		std::string logfileNamePrefix(options.get("logfile"));
 		Log::global_log->info() << "Using logfile with prefix " << logfileNamePrefix << std::endl;
-		delete Log::global_log;
-		Log::global_log = new Log::Logger(Log::Info, logfileNamePrefix);
+		Log::global_log.reset();
+		Log::global_log = std::make_unique<Log::Logger>(Log::Info, logfileNamePrefix);
```

Comment:
I find it VERY unintuitive that this switch to the logfile comes only after a few messages have already been issued. Especially the version messages are quite important. Can we move this part further up so that the logger is only initialized once and then directly with the desired output channel?